AirCasting/SessionsSynchronization/Controller/SessionSynchronizationController.swift

```diff
@@ -5,15 +5,18 @@ import Foundation
 import Combine
 import CoreLocation
 
+public var isSyncing = false
```

Comment:
no no, this cannot be done this way. Consider global vars as strictly forbidden

Reply:
I thought that You will say something like this 😅. Could You please tell me, give me some clue how to do it well?